third_party/phantomjs/phantomjs.sh

```diff
@@ -19,11 +19,14 @@
 #         export FC_DEBUG="1024"
 
 if [[ "${RUNFILES}" == "" ]]; then
-  if [[ "$(basename "$(pwd)")" =~ \.runfiles$ ]]; then
-    RUNFILES="$(pwd)"
+  if [[ "$(pwd)" =~ ^(.*\.runfiles)(/.*)?$ ]]; then
+    RUNFILES="${BASH_REMATCH[1]}"
   else
     RUNFILES="${0}.runfiles"
   fi
+  if [ -d "${RUNFILES}/io_bazel_rules_closure" ]; then
```

Comment:
1. Since I set the workspace(name), wouldn't this *always* be true? If so, why would we have an if statement?
2. Where will external dependency runfiles be stored? Will it be io_bazel_rules_closure/external/phantomjs? Or external/phantomjs?
3. If it's the prior, then I'm assuming Bazel now transitively fetches WORKSPACE dependencies? If so, should the contents of my repositories.bzl be moved into my WORKSPACE file?